config/foreman.hiera/custom.yaml

```diff
@@ -15,3 +15,14 @@
 # performance or functionality of the application. Consider the impact of your
 # changes before applying them, and test them in a non-production environment
 # first.
+#
+# Here are some examples of how you tune the Apache and Passenger options if needed:
+#
+# apache::mod::passenger::passenger_max_preloader_idle_time: 0
+# apache::mod::passenger::passenger_passenger_start_timeout: 500
```

Comment:
I think this parameter doesn't make sense anymore. puppet-foreman sets it explicitly and exposes it at a top level parameter.